### ISSUE TITLE
Extension causes menu and toolbox in draw.io / diagrams.net to disappear

### ISSUE THREAD

**Issue by sebastian-fredriksson-bernholtz** [igrigorik/videospeed#966]
See below screenshots. The only difference between them is whether the this extension is enabled or disabled. Let me know if there is any more information you want me to provide.

<img width="1792" alt="Screenshot 2022-09-17 at 10 11 37" src="https://user-images.githubusercontent.com/14281373/190832500-b7486c8c-5bcf-4be8-9096-4657cc2dadad.png">
<img width="1792" alt="Screenshot 2022-09-17 at 10 11 19" src="https://user-images.githubusercontent.com/14281373/190832501-b0b5d0f8-6d4d-412e-a44a-f0f08ea4d56c.png">

**Comment by ChadBailey** [comment]
Does disabling via the options menu fix the problem? If so, would adding it to the default list of blocked sites suffice?

**Comment by thewheat** [comment]
I can't seem to replicate the issue with my machine

chrome://version/
```
Google Chrome 106.0.5249.91 (Official Build) (x86_64)  
OS macOS Version 12.6 (Build 21G115)
```

chrome://extensions/?id=nffaoalbilbmmfgbnbgppjihopabppdk
```
VSC version 0.7.1
```

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/892961/194053685-871a7501-9d52-421e-a9ff-610c6b57c5cb.png">

@sebastian-fredriksson-bernholtz could you try again to see if it is still an issue? 

- Are there any browser / OS / VSC version differences with my tests below
- Perhaps you could try using Incognito mode with only VideoSpeedController enabled to see if it could be conflicting with other extensions

**Comment by sebastian-fredriksson-bernholtz** [comment]
Sorry for delayed response. I should have included version information. I have since updated Mac OS and Chrome versions and it's working as expected now. Not sure if it's the update or maybe just that I've since restarted my browser.

Sorry for wasting your time on this. Thanks for an amazing extension! 🎉

**Comment by silentbrains** [comment]
I also noticed this problem, but it went away on its own so I forgot about it. Did some digging, and it seems it was fixed in drawio 20.3.3?
https://github.com/jgraph/drawio/issues/3061
